<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <widget class="QMenu" name="menuIstorija">
     <property name="title">
      <string>Istorija</string>
     </property>
     <addaction name="actionIzlistaj"/>
     <addaction name="actionOcisti"/>
     <addaction name="actionUcitaj_novu"/>
    </widget>
    <addaction name="actionPodesavanja_2"/>
    <addaction name="menuIstorija"/>
    <addaction name="separator"/>
    <addaction name="actionNapusti_program"/>
   </widget>
   <widget class="QMenu" name="menuPogledi">
    <property name="title">
     <string>Pogledi</string>
    </property>
    <addaction name="actionCeline"/>
   </widget>
   <widget class="QMenu" name="menuSkoci_na">
    <property name="title">
     <string>Skoci</string>
    </property>
    <addaction name="actionUvod"/>
    <addaction name="actionEditor"/>
    <addaction name="actionAutomat"/>
    <addaction name="actionDiagram"/>
   </widget>
   <widget class="QMenu" name="menuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="actionPodrska"/>
    <addaction name="separator"/>
    <addaction name="actionAutorska_prava"/>
    <addaction name="actionPravila_koriscenja"/>
    <addaction name="separator"/>
    <addaction name="actionT_Rex_Fb"/>
    <addaction name="actionT_Rex_Twitter"/>
    <addaction name="actionT_Rex_GitHub"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_T_Rex"/>
   </widget>
   <addaction name="menuProgram"/>
   <addaction name="menuPogledi"/>
   <addaction name="menuSkoci_na"/>
   <addaction name="menuPomoc"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPozovi">
   <property name="text">
    <string>Pozovi</string>
   </property>
  </action>
  <action name="actionIzlistaj">
   <property name="text">
    <string>Izlistaj</string>
   </property>
  </action>
  <action name="actionOcisti">
   <property name="text">
    <string>Ocisti</string>
   </property>
  </action>
  <action name="actionNapusti_program">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/menus/exit.png</normaloff>:/images/menus/exit.png</iconset>
   </property>
   <property name="text">
    <string>Napusti program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Celine</string>
   </property>
  </action>
  <action name="actionUvod">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icon/uvod.png</normaloff>:/images/icon/uvod.png</iconset>
   </property>
   <property name="text">
    <string>Uvod</string>
   </property>
  </action>
  <action name="actionAutomat">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icon/automat.png</normaloff>:/images/icon/automat.png</iconset>
   </property>
   <property name="text">
    <string>Automat</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icon/editor.png</normaloff>:/images/icon/editor.png</iconset>
   </property>
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionDiagram">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/icon/diagram.png</normaloff>:/images/icon/diagram.png</iconset>
   </property>
   <property name="text">
    <string>Diagram</string>
   </property>
  </action>
  <action name="actionSrpski">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Srpski</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionPodesavanja">
   <property name="text">
    <string>Podesavanja</string>
   </property>
  </action>
  <action name="actionPodrska">
   <property name="text">
    <string>Podrska</string>
   </property>
  </action>
  <action name="actionPravila_koriscenja">
   <property name="text">
    <string>Pravila koriscenja</string>
   </property>
  </action>
  <action name="actionAutorska_prava">
   <property name="text">
    <string>Autorska prava</string>
   </property>
  </action>
  <action name="actionT_Rex_Fb">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/menus/fb_icon.png</normaloff>:/images/menus/fb_icon.png</iconset>
   </property>
   <property name="text">
    <string>T-Rex: Facebook</string>
   </property>
  </action>
  <action name="actionT_Rex_Twitter">
   <property name="text">
    <string>T-Rex: Twitter</string>
   </property>
  </action>
  <action name="actionAbout_T_Rex">
   <property name="text">
    <string>About T-Rex</string>
   </property>
  </action>
  <action name="actionT_Rex_GitHub">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/menus/code_i1.png</normaloff>:/images/menus/code_i1.png</iconset>
   </property>
   <property name="text">
    <string>T-Rex: GitHub</string>
   </property>
  </action>
  <action name="actionBroj_prepoznatih">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Broj prepoznatih</string>
   </property>
  </action>
  <action name="actionUcitaj_novu">
   <property name="text">
    <string>Ucitaj novu</string>
   </property>
  </action>
  <action name="actionSimulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulator</string>
   </property>
  </action>
  <action name="actionPodesavanja_2">
   <property name="text">
    <string>Podesavanja</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
